torch/csrc/jit/passes/utils/subgraph_utils.cpp

```diff
@@ -26,9 +26,11 @@ std::vector<c10::optional<const Use>> gatherLastUses(
 // Values which do not have uses or which do not have a last use
 // outside of the subgraph to be merged into we do not need to track.
 struct ValueMapper {
-  ValueMapper(Node* to_merge, AliasDb& db, size_t subgraph_num_outputs) {
+  ValueMapper(Node* to_merge, AliasDb& db, c10::optional<Node*> existing) {
```

Comment:
could we relate the comment below to `ValueMapper` c-tor's parameters?

Reply:
i'm not sure what you mean

Reply:
I'm not sure what @Krovatkin meant, but I'd find it helpful if we had a description of what each argument of the constructor is used for (especially the last one).